apollo-adminservice/src/main/java/com/ctrip/framework/apollo/adminservice/aop/NamespaceUnlockAspect.java

```diff
@@ -33,21 +32,16 @@
 import com.google.common.collect.MapDifference;
 import com.google.common.collect.Maps;
 import com.google.gson.Gson;
+import java.util.List;
+import java.util.Map;
 import org.aspectj.lang.annotation.After;
 import org.aspectj.lang.annotation.Aspect;
 import org.springframework.stereotype.Component;
 
-import java.util.List;
-import java.util.Map;
-
-
 /**
- * unlock namespace if is redo operation.
- * --------------------------------------------
- * For example: If namespace has a item K1 = v1
- * --------------------------------------------
- * First operate: change k1 = v2 (lock namespace)
- * Second operate: change k1 = v1 (unlock namespace)
+ * unlock namespace if is redo operation. -------------------------------------------- For example:
+ * If namespace has a item K1 = v1 -------------------------------------------- First operate:
+ * change k1 = v2 (lock namespace) Second operate: change k1 = v1 (unlock namespace)
```

Comment:
This also doesn't look good.

Reply:
Use args `--skip-javadoc-formatting` to avoid it.